src/thoughts-on/escaping-the-labyrinth.html

```diff
@@ -0,0 +1,610 @@
+        <p>
+          How does Ariadne's Thread work in practice? Here's a step-by-step
+          guide to creating a thread for a small project:
+        </p>
+        <ul>
+          <li>Create your project (I use VS Code)</li>
+          <li>Add an artefacts folder</li>
+          <li>
+            Create a new thread prompt in artefacts with project details,
+            interaction rules and how you want the LLM to write snapshots
+          </li>
+          <li>Open a new chat and upload the thread</li>
+          <li>
+            Add any files used during the interaction to artefacts, including
+            those generated by the LLM
+          </li>
+          <li>
+            When you need a summary of progress to date, ask for a snapshot and
+            add it to the snapshots array
+          </li>
+          <li>
+            Use the snapshots and LLM responses inside artefacts to think over
+            what you are doing before working on source code away from the
+            distractions of a chat interface
+          </li>
+        </ul>
+        <aside>
+          <p>
+            The Ariadne's Thread repo has a visual representation of the
+            <a
+              href="https://github.com/p-n-c/ariadnes-thread/blob/main/WORKFLOW.md"
+              >workflow</a
+            >.
+          </p>
+        </aside>
+        <section aria-labelledby="example">
+          <h4 id="example">
+            Example - Building a simple RSS feed generator with Claude
+          </h4>
+          <p>
+            I used Ariadne's Thread to create a simple RSS feed generator. In
+            the
+            <a
+              href="https://github.com/p-n-c/rss-feed-generator/tree/main/artefacts"
+              >artefacts</a
+            >
+            folder you will find a thread, an example of an RSS feed which I
+            added to the chat, and an rss-generator file with code from the LLM.
+            Because I had asked Claude to proceed incrementally, it returned
+            small chunks of code (numbered 1-4) which I evaluated and refactored
+            before using in the source code.
+          </p>
+        </section>
+      </section>
+      <section aria-labelledby="conclusion">
+        <h2 id="conclusion">Conclusion</h2>
+        <p>
+          In order not to get lost in the maze of dead-ends and wrong turns
+          caused by muddled thinking, I struck on the metaphor of the thread
+          which Ariadne gave to Theseus to help him navigate the labyrinth.
+        </p>
+        <p>
+          Creating a thread has the effect of making me think more carefully
+          before starting a new project. Its structure and documentation afford
+          me better oversight and - if I ask an LLM to assist me - the means to
+          moderate our interaction.
+        </p>
+        <aside>
+          <p>
+            The
+            <a href="https://github.com/p-n-c/project-prototype-generator"
```

Comment:
👍 - I would also/alternatively point to it in the workflow, where people may need it.